src/drivers/intel/cavs/ipc.c

```diff
@@ -131,7 +131,7 @@ static enum task_state ipc_platform_do_cmd(void *data)
 	ipc_cmd(hdr);
 
 	/* are we about to enter D3 ? */
-#if CAVS_VERSION < CAVS_VERSION_2_0
+#if !defined CONFIG_SUECREEK
```

Comment:
#ifs on lines 146 and 165 are now only needed for `CONFIG_SUECREEK`.

Reply:
ACK